### ISSUE TITLE
HomeKit Secure Video

### ISSUE THREAD

**Issue by F-John** [hjdhjd/homebridge-unifi-protect#232]
**Is your feature request related to a problem? Please describe:**
Thank you @hjdhjd for this wonderful repo! I have been using this from the very beginning of my home upgrade from consumer grade networking upgrade to Unifi and Protect. It has always worked exactly as expected, which is less than I can say for Unifi's own AppleTV app.

**Describe the solution you'd like:**
Do you have any plans to bring in HomeKit Secure Video support?

This allows video backup to go to iCloud and KomeKit users that have recording access to view recorded events from the cloud which lets us skip having to invite users to the Unifi Protect environment, and just give them access to the Secure Video recordings directly from the Home app.

**Describe alternatives you've considered:**
Really only wondering if HomeKit Secure Video (HSV) is on your radar or roadmap.

**Additional context:**
Secure Video Specification:
https://github.com/Supereg/secure-video-specification

**Comment by hjdhjd** [comment]
Your welcome and happy new year!

This is on the radar, but it’s got a long, long way to go. I’ve said this before, as recently as three days ago:

https://github.com/hjdhjd/homebridge-unifi-protect/issues/226#issuecomment-752728445

TL;DR: would love to support it, but it’s going to be a while until all the infrastructure exists in Homebridge to support it. We’re actively working on it as a community.

**Comment by F-John** [comment]
Copy, understood. Thank you for your fast response and Happy New Year to you too.

**Comment by hjdhjd** [comment]
Thanks for the support and the feedback. Appreciate it...and as soon as it’s feasible to add in support for HSV, I will. It’s a journey, like many things HomeKit-related. :smile:
